sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1210,20 +1236,32 @@ def process(self, element, *schema_side_inputs):
       return self._flush_all_batches()
 
   def finish_bundle(self):
-    if BigQueryWriteFn.LATENCY_LOGGING_LOCK.acquire(False):
+    if BigQueryWriteFn.STREAMING_API_LOGGING_LOCK.acquire(False):
       try:
         current_millis = int(time.time() * 1000)
-        if (BigQueryWriteFn.LATENCY_LOGGING_HISTOGRAM.total_count() > 0 and
-            (current_millis -
-             BigQueryWriteFn.LATENCY_LOGGING_LAST_REPORTED_MILLIS) >
-            self._latency_logging_frequency_msec):
-          _LOGGER.info(
-              BigQueryWriteFn.LATENCY_LOGGING_HISTOGRAM.get_percentile_info(
-                  'streaming insert requests', 'ms'))
-          BigQueryWriteFn.LATENCY_LOGGING_HISTOGRAM.clear()
-          BigQueryWriteFn.LATENCY_LOGGING_LAST_REPORTED_MILLIS = current_millis
+        if ((current_millis -
+             BigQueryWriteFn.STREAMING_API_LOGGING_LAST_REPORTED_MILLIS) >
+            self._streaming_api_logging_frequency_msec):
+          streaming_api_info = [
+              '[Streaming Insert API Statistics since %s]' %
+              datetime.datetime.fromtimestamp(
+                  BigQueryWriteFn.STREAMING_API_LOGGING_LAST_REPORTED_MILLIS /
+                  1000.0)
+          ]
+          if BigQueryWriteFn.LATENCY_LOGGING_HISTOGRAM.total_count() > 0:
+            streaming_api_info.append(
```

Comment:
Please address code coverage warnings


Codecov
/ codecov/patch

sdks/python/apache_beam/io/gcp/bigquery.py#L1252
Added line #L1252 was not covered by tests

Reply:
done.